src/puppetlabs/trapperkeeper/services/status/status_logging.clj

```diff
@@ -0,0 +1,22 @@
+(ns puppetlabs.trapperkeeper.services.status.status-logging
+  (:require [clojure.tools.logging :as log]
+            [schema.core :as schema]
+            [schema.utils :refer [validation-error-explain]]
+            [cheshire.core :as json]
+            [puppetlabs.trapperkeeper.services.status.status-core :as status-core])
+  (:import (clojure.lang IFn)
+           (java.util.concurrent Future)))
+
+(schema/defn start-background-task :- Future
+  "Creates a future which calls callback every frequency milliseconds"
+  [frequency :- schema/Int
+   callback :- IFn]
+  (future (while true
+            (callback)
+            (Thread/sleep frequency))))
+
+(defn logging-fn
+  "Log status information at the debug level as json"
+  []
+  (let [status (status-core/v1-status :debug)]
```

Comment:
Derp - I'm an idiot, that's not a logging level above.  So the actual logging level is `info`, below.  In that case it seems like you definitely would have seen these messages in the root logger if that config wasn't working the way we wanted?  So, that's good?  I do think you probably want to log these at `debug`, though?